src/index.html

```diff
@@ -7,8 +7,59 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/174788310-d64c6f75-6b67-4c9e-ab1b-ed515017bcfe.png)
This blue line should be perfectly aligned with the bottom side of the header, for now I can see some space between them

Reply:
![image](https://user-images.githubusercontent.com/38065883/177157885-6b1659cb-ac13-4210-ae07-ee248d2dc38b.png)
Have you deployed last version? there is still space between link and bottom edge of the header?